src/curves/curve.ts

```diff
@@ -18,13 +18,13 @@ export type Curve = {
     n: bigint;
   };
   getPublicKey: (
-    privateKey: Uint8Array | string | bigint,
```

Comment:
Why are we limiting support here? 🤔

Reply:
It looks like the underlying implementation doesn't support `string` or `bigint`? I was running into errors, even though the type says it's supported. Regardless, it's better to just accept `Uint8Arrays`, as otherwise we run into the same problem as mentioned above for string values.